### ISSUE TITLE
Question about card images

### ISSUE THREAD

**Issue by wzurborg** [mitch-b/typedeck#35]
Hi & thank you for this library!

In the Readme you give credits for card images but there are no images anywhere in this project. Is that a leftover?

What is the recommended way of adding card images? There is no implementation of ICardImageService either, should I start by writing an own implementation?

**Comment by mitch-b** [comment]
Yes, in the README, there's an Ace of Diamonds card that's actually an SVG from thenounproject - [here](https://thenounproject.com/search/?creator=64420&q=diamond&i=764379). 

I feel like `ICardImageService` is either:
1. A nice idea that I never got around to completely implementing. 
    * I'm not 100% sure the interface completely covers use cases yet, as I don't think I ever saw it through with a sample implementation myself. 
 2. Unless bundled with the library - an implementation will have an unknown external dependency for images that would be hard to manage. 
     * Or, if a default was provided, file size of package would increase for everyone.
 
I'm curious what you have in mind, though!

**Comment by wzurborg** [comment]
I think for the start and for simplicity I'll just keep the card images in my local project assets. I can always switch to multi-themes and web based storage later.

One more thing: I am a very old programmer but new to npm. So I cloned the Git repository locally and tried a lot of things to make it available as a dependency of my project. After many fruitless attempts I desperately entered "npm install typedeck" and it was there! Magic! One sentence in the Readme about the installation would perhaps save much frustration.

Cheers, Wernke
